packages/dataviews/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking changes
 
 - DataForm: Add summary field support for both card and panel layouts. The `summary` property has been moved from the field level to the layout level, and so fields using `summary` at the field level must now configure it within the `layout` object. Additionally, the first children will only be used as summary for the panel if 1) there is no `layout.summary` and 2) the form field ID doesn't match any existing field. See README for details. [#71576](https://github.com/WordPress/gutenberg/pull/71576)
+- Remove `Data< Item >` type, as it is no longer used internally for a long time.
```

Comment:
Wasn't sure if we need to communicate type changes here, but given there's already a breaking change section I thought it wouldn't hurt.